cdap-docs/build.sh

```diff
@@ -149,11 +153,10 @@ function build_docs_outer_level() {
   echo "========================================================"
   echo ""
 
-  copy_html admin-manual
-  copy_html developers-manual
-  copy_html integrations
-  copy_html reference-manual
-  copy_html examples-manual
+  for i in "${manuals[@]}"
```

Comment:
``for i in ${MANUALS}`` would work too.  No need to have converted it to an array